apps/ashley/templates/services/nginx/ingress.yml.j2

```diff
@@ -12,7 +12,7 @@ metadata:
     route_prefix: "{{ prefix }}"
     route_target_service: "app"
   annotations:
-    cert-manager.io/cluster-issuer: "{{ acme_issuer_name }}"
+    cert-manager.io/issuer: "{{ acme_issuer_name }}"
```

Comment:
Should it be configurable?

Reply:
Since arnold explicitly creates an Issuer per namespace during the project bootstrap, I don't think this is necessary in the current state of the project.

Should it be configurable per project, or across all arnold apps ?
It might be better to develop this in another PR, if we want to do it.

Reply:
ok then